easybuild/easyconfigs/l/longread_umi/longread_umi-0.3.2-foss-2020b.eb

```diff
@@ -0,0 +1,79 @@
+easyblock = 'Bundle'
+
+name = 'longread_umi'
+version = '0.3.2'
+
+homepage = 'https://github.com/SorenKarst/longread_umi'
+description = "A collection of scripts for processing longread UMI data."
+github_account = 'SorenKarst'
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('cutadapt', '3.4'),
+    ('BCFtools', '1.11'),
+    ('BWA', '0.7.17'),
+    ('Filtlong', '0.2.0'),
+    ('gawk', '5.1.0'),
+    ('medaka', '1.4.3'),
+    ('minimap2', '2.18'),
+    ('parallel', '20210322'),
+    ('Pysam', '0.16.0.1'),
+    ('Racon', '1.4.13'),
+    ('SAMtools', '1.11'),
+    ('seqtk', '1.3'),
+    ('VSEARCH', '2.18.0'),
+]
+
+components = [
+    (name, version, {
+        'easyblock': 'Tarball',
+        'source_urls': [GITHUB_LOWER_SOURCE],
+        'sources': ['v%(version)s.tar.gz'],
+        'checksums': ['62b8e156c00c0ec10fa8eae1cde5430922462f167fc537417ce0b47cd50a20cb'],
+        'start_dir': '%(name)s-%(version)s',
+    }),
+    ('Porechop', '0.2.4', {
+        'easyblock': 'PythonPackage',
+        'source_urls': ['https://github.com/rrwick/Porechop/archive'],
+        'sources': ['v%(version)s.tar.gz'],
+        'checksums': ['44b499157d933be43f702cec198d1d693dcb9276e3c545669be63c2612493299'],
+        'start_dir': '%(name)s-%(version)s',
+        'use_pip': True,
+        'download_dep_fail': True,
+        'sanity_pip_check': True,
+    }),
+]
+
+# Adapt the build-in tool check, but make it fail on error
```

Comment:
this patch command does more: also switches to `vsearch`, and fixes the version bit